charters/UNITY/charter.md

```diff
@@ -0,0 +1,54 @@
+# Unity Benchmarking Service
```

Comment:
Is _Unity_ a codename or an aspiration?

Reply:
UNITY is the name of the service, the aspiration, and a part of the international space station (where we get many of our names, eg Skylab). For clarity, the directory can be called UnityBenchMarkingService.